.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-    - '0.10'
+    - '5'
```

Comment:
Update the readme or something to note this requirement due to es6 in use in Gruntfile